flake-module.nix

```diff
@@ -6,13 +6,15 @@ let
     mdDoc
     mkOption
     types
-    literalExpression;
+    literalExpression
+    warn;
 in
 {
   options = {
     perSystem = mkPerSystemOption
       ({ config, self', inputs', pkgs, system, ... }: {
         options.process-compose = mkOption {
+          default = warn "process-compose flake module was imported but not used" { };
```

Comment:
@roberth can corroborate; I think it is more idiomatic to initialize the top-level config to an empty attrset (#8) rather than throw a warning.

Reply:
How about an option for linting messages that don't warn by default?
I could add that to the debug attribute in flake-parts. That way we can figure some things out for the user without having to bother them with warnings which may or may not apply.

Reply:
You could try the `lint` options implemented in https://github.com/hercules-ci/flake-parts/pull/134

Docs:

    nix build github:hercules-ci/flake.parts-website --override-input flake-parts flake-parts/lint \
      && xdg-open result/options/flake-parts.html

Reply:
I think having some kind of information is neat because it makes you aware that you have something in your flake inputs you don't actually use. I chose to warn because it was the closest thing I found. Defaulting to an empty attribute-set makes it difficult to tell right away whether you use it or not in a big project. Also the warning does not make your build fail.

Reply:
I would be happy if flake-part itself would inform me about unused modules.